### DISCUSSION TITLE
Pocketbase Trigger (onRecordAfterCreateRequest)

### DISCUSSION THREAD

**Discussion by amirkhalil22** [pocketbase/pocketbase#3349] [Q&A] [answered]
i want to create a record AfterCreateRequest, so notifications is the collection where that record will be created with the trigger and bulletins is the collection where i will create my record, so bulletins have field named clients (multiple relation with collection named clients) and notifications has also field named cli(multiple relation with clients "the same as bulletins"), i can send the other field like a boolean or single relation likee the picture below but i can't do it with this multiple field 

please any help ??

![image](https://github.com/pocketbase/pocketbase/assets/104743918/195eb958-8a5a-48ff-86bf-b6af12864fef)

**Comment by ganigeorgiev** [comment]
`expandedAll` will return the expand result **only** if the expand data was already previously fetched/loaded.

To do that you can check the examples in https://pocketbase.io/docs/js-records/#programmatically-expanding-relations.

But if you just want the client ids, then I don't think you need the expand data and you should be able to use directly:

```js
const clientIds = e.record.get("clients");
```

**Reply by amirkhalil22** [reply]
and i must doing it with .map or the cli field can take clientIds directly ???

**Reply by ganigeorgiev** [reply, marked as answer]
I'm not sure I follow.
You don't need `map` in this case because `e.record.get("clients")` should return an array of the client ids.
Alternatively you can also "force" the casting using `e.record.getStringSlice("clients")`

**Reply by amirkhalil22** [reply]
thanks a lot 🥇
